mkt/webapps/management/commands/process_addons.py

```diff
@@ -23,41 +18,11 @@
                                               mkt.STATUS_PUBLIC,
                                               mkt.STATUS_APPROVED],
                                   disabled_by_user=False)]},
-    'add_uuids': {'method': add_uuids,
-                  'qs': [Q(guid=None), ~Q(status=mkt.STATUS_DELETED)]},
     'update_supported_locales': {
         'method': update_supported_locales,
         'qs': [Q(disabled_by_user=False,
                  status__in=[mkt.STATUS_PENDING, mkt.STATUS_PUBLIC,
                              mkt.STATUS_APPROVED])]},
-    'dump_apps': {'method': dump_apps,
-                  'qs': [Q(status=mkt.STATUS_PUBLIC, disabled_by_user=False)],
-                  'pre': clean_apps,
-                  'post': zip_apps},
-    'fix_missing_icons': {'method': fix_missing_icons,
```

Comment:
One-shot command used to work around issues with the icon task being called too soon a while ago. Was fixed when we enabled transaction middleware everywhere and made sure to trigger tasks after the request/response cycle is done. AFAIK we haven't used it since.